### PR TITLE
CSP-1791 remove kibana logging

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.Shortlist.Api/appsettings.Development.json
Comment (DevenKShah) [inline src/SFA.DAS.Shortlist.Api/appsettings.Development.json:1]: I think we should delete this as well.

**Review by DevenKShah — APPROVED** [review]

File: src/SFA.DAS.Shortlist.Api/Program.cs [hunk 1]
```diff
@@ -50,6 +47,9 @@ public static void Main(string[] args)
             builder.Services.AddValidatorsFromAssembly(typeof(ShortlistAddModelValidator).Assembly);
             builder.Services.AddFluentValidationAutoValidation();
 
+            builder.Services.AddLogging()
```
Comment (DevenKShah) [inline src/SFA.DAS.Shortlist.Api/Program.cs:50]: This is missing adding azure monitor bit

File: src/SFA.DAS.Shortlist.Api/AppStart/AddTelemetryRegistrationExtension.cs
Comment (DevenKShah) [inline src/SFA.DAS.Shortlist.Api/AppStart/AddTelemetryRegistrationExtension.cs:1]: Why do we need two extension methods, this is just calling the other one, perhaps we can remove this one.

File: README.md [hunk 2]
```diff
@@ -12,6 +12,24 @@ _Update these badges with the correct information for this project. These give t
...
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConfigurationStorageConnectionString": "UseDevelopmentStorage=true;",
+  "ConfigNames": "SFA.DAS.Shortlist.Api",
+  "Environment": "LOCAL",
+  "Version": "1.0",
+  "APPINSIGHTS_INSTRUMENTATIONKEY": ""
```
Comment (DevenKShah) [inline README.md:29]: This should be replaced with APPLICATIONINSIGHTS_CONNECTION_STRING

**Review by PeterDavidWilliams — APPROVED** [review]

**Review by shaun-downey-education — APPROVED** [review]

**Pull request merged**